src/components/about/About.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Footer from '../footer/Footer.tsx'
+import Nav from './../nav/Nav.tsx'
```

Comment:
Let's change the relative import to absolute.
an import like '../../../' looks a bit odd... and when someone reads the code, it might not be clear what directory it comes from and so on...